Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -1,7 +1,104 @@
 ﻿using System;
-
+using System.Text.RegularExpressions;
 namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
 {
+    class Equation{
+        public Equation(string operand1,string operand2,string operation,string result){
+            this.operand1=operand1;
+            this.operand2=operand2;
+            this.operation=operation;
+            this.result=result;
+        }
+        public enum EquationParts{
+            operand1,operand2,result

```

Comment:
Operand1, Operand2, Result.
These are members of Enum, first character should be in upper case